src/lib/message.ts

```diff
@@ -1,6 +1,32 @@
 import EventEmitter from 'events'
+import GleeConnection from './connection'
+
+interface IGleeMessageConstructor {
+  payload?: any,
+  headers?: Map<string, string>,
+  channel?: string,
+  serverName?: string,
+  connection?: GleeConnection,
+  broadcast?: boolean,
+}
+
+interface IReply {
+  payload?: any,
+  headers?: Map<string, string>,
+  channel?: string,
+}
 
 class GleeMessage extends EventEmitter {
+  public payload: any
+  public headers: Map<string, string>
+  public channel: string
+  public serverName: string
+  public connection: GleeConnection
+  public broadcast: boolean
+  public __isGleeMessage: boolean
+  public inbound: boolean
+  public outbound: boolean
```

Comment:
I'll make this stuff private in the next PR. I want to leave it public so it behaves the same way as in JS.

Reply:
https://github.com/asyncapi/glee/pull/213/files#diff-69436b7a8068a847056d1257334d0e1c90e2873bafc8b4a9bcd9db9b6bdfa11bR20-R27